src/js/main.js

```diff
@@ -169,7 +169,7 @@ $(document).ready(function() {
       }
       if (!jeepImageCompleted) {
         if (scrollActivationPoint > jeepImageOffset) {
-          $('#jeepImage').addClass('active');
+          $('#jeep-image').addClass('active');
```

Comment:
Please always dasherise HTML IDs. You were right to camelcase JS variables though.